### PR TITLE
Add More Imagenet Demo

### PR DESCRIPTION
**Pull request by RachelXu7**

### CODE REVIEW

File: demo/auto_compression/image_classification/configs/EfficientNetB0/.ipynb_checkpoints/prune_dis-checkpoint.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+Global:
```
Comment (ceci3) [inline demo/auto_compression/image_classification/configs/EfficientNetB0/.ipynb_checkpoints/prune_dis-checkpoint.yaml]: 这些临时文件清理一下？

File: demo/auto_compression/image_classification/README.md [hunk 2]
```diff
@@ -16,14 +16,38 @@
...
+| GhostNet_x1_0 | 量化+蒸馏 | 71.11 | 1.0348 | - |
+| InceptionV3 | Baseline | 79.14 | 4.7931 | - |
+| InceptionV3 | 量化+蒸馏 | 73.16 | 1.4703 | - |
+| MobileNetV3_large_x1_0 | Baseline | 75.32 | - | 16.62 |
+| MobileNetV3_large_x1_0 | 量化+蒸馏 | 68.84 | - | 9.85 |
+
+- ARM CPU 测试环境：`SDM710 2*A75(2.2GHz) 6*A55(1.7GHz)`
+- Nvidia GPU 测试环境：
+  - 硬件：NVIDIA Tesla T4 单卡
+  - 软件：CUDA 11.0, cuDNN 8.0, TensorRT 8.0
```
Comment (ceci3) [inline demo/auto_compression/image_classification/README.md]: CUDA 11.2, cuDNN 8.0, TensorRT 8.4

File: demo/auto_compression/image_classification/README.md [hunk 3]
```diff
@@ -16,14 +16,38 @@
...
+| EfficientNetB0 | Baseline | 77.02 | 1.9529 | - |
+| EfficientNetB0 | 量化+蒸馏 | 73.61 | 1.4362 | - |
+| GhostNet_x1_0 | Baseline | 74.02 | 2.9301 | - |
+| GhostNet_x1_0 | 量化+蒸馏 | 71.11 | 1.0348 | - |
+| InceptionV3 | Baseline | 79.14 | 4.7931 | - |
+| InceptionV3 | 量化+蒸馏 | 73.16 | 1.4703 | - |
+| MobileNetV3_large_x1_0 | Baseline | 75.32 | - | 16.62 |
+| MobileNetV3_large_x1_0 | 量化+蒸馏 | 68.84 | - | 9.85 |
+
+- ARM CPU 测试环境：`SDM710 2*A75(2.2GHz) 6*A55(1.7GHz)`
```
Comment (ceci3) [inline demo/auto_compression/image_classification/README.md]: 再确认下ARM CPU的环境？

File: demo/auto_compression/image_classification/README.md [hunk 4]
```diff
@@ -71,24 +95,11 @@ tar -xf MobileNetV1_infer.tar
 ```shell
 # 单卡启动
 export CUDA_VISIBLE_DEVICES=0
```
Comment (ceci3) [inline demo/auto_compression/image_classification/README.md:110]: 单卡启动示例没有了？

File: demo/auto_compression/image_classification/README.md [hunk 5]
```diff
@@ -71,24 +95,11 @@ tar -xf MobileNetV1_infer.tar
...
-    --model_filename='inference.pdmodel' \
-    --params_filename='inference.pdiparams' \
-    --save_dir='./output' \
-    --batch_size=128 \
-    --config_path='./configs/mobilenetv1_qat_dis.yaml'\
-    --data_dir='ILSVRC2012'
+export CUDA_VISIBLE_DEVICES=0,1,2,3
+
+python run.py --save_dir='./save_quant_mobilev1/' --config_path='./configs/MobileNetV1/qat_dis.yaml'
+
```
Comment (ceci3) [inline demo/auto_compression/image_classification/README.md:115]: 多卡启动示例不对

File: demo/auto_compression/image_classification/README.md [hunk 6]
```diff
@@ -99,4 +110,3 @@ python -m paddle.distributed.launch run.py \
 - [Paddle Lite部署](https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.5/docs/deployment/lite/lite.md)
 
 ## 5.FAQ
```
Comment (ceci3) [inline demo/auto_compression/image_classification/README.md:125]: FAQ为啥删掉了？

File: demo/auto_compression/image_classification/configs/GhostNet_x1_0/prune_dis.yaml [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+  data_dir: /ILSVRC2012
+  
+Distillation:
+  alpha: 1.0
+  loss: l2
+  node:
+  - softmax_0.tmp_0
+  teacher_model_dir: GhostNet_x1_0_infer
+  teacher_model_filename: inference.pdmodel
+  teacher_params_filename: inference.pdiparams
```
Comment (ceci3) [inline demo/auto_compression/image_classification/configs/GhostNet_x1_0/prune_dis.yaml]: teacher_model_dir， teacher_model_filename，teacher_params_filename这三个可以不写

File: demo/auto_compression/image_classification/run.sh [hunk 8]
```diff
@@ -1,21 +1,9 @@
...
-    --model_dir='MobileNetV1_infer' \
-    --model_filename='inference.pdmodel' \
-    --params_filename='inference.pdiparams' \
-    --save_dir='./save_quant_mobilev1/' \
-    --batch_size=128 \
-    --config_path='./configs/mobilev1.yaml'\
-    --data_dir='ILSVRC2012'
+# 多卡启动
+# export CUDA_VISIBLE_DEVICES=0,1,2,3
+python run.py --save_dir='./save_quant_mobilev1/' --config_path='./configs/MobileNetV1/qat_dis.yaml'
```
Comment (ceci3) [inline demo/auto_compression/image_classification/run.sh]: 多卡启动命令不对

**Review by ceci3 — COMMENTED** [review]
不同模型如果使用相同的yaml文件可以公用一份

File: demo/auto_compression/image_classification/README.md (hunk 4 above)
Comment (RachelXu7) [reply demo/auto_compression/image_classification/README.md:110]: 单多卡都一样 export几个就是几个

File: demo/auto_compression/image_classification/README.md (hunk 6 above)
Comment (RachelXu7) [reply demo/auto_compression/image_classification/README.md:125]: 要求输入input name

File: demo/auto_compression/image_classification/README.md (hunk 4 above)
Comment (ceci3) [reply demo/auto_compression/image_classification/README.md:110]: 多卡得用 distributed.launch的方式启动

Comment (RachelXu7) [reply demo/auto_compression/image_classification/README.md:110]: 不用。。

File: demo/auto_compression/image_classification/README.md [hunk 9]
```diff
@@ -16,33 +16,38 @@
...
-| MobileNetV1 | Base模型 | 71.0 | - | [Model]() |
-| MobileNetV1 | 量化+蒸馏 | 70.22 | -| [Model]() |
-
-- 测试环境：
-
-说明：
-- MobileNetV1模型源自[tensorflow/models](http://download.tensorflow.org/models/mobilenet_v1_2018_02_22/mobilenet_v1_1.0_224.tgz)，通过[X2Paddle](https://github.com/PaddlePaddle/X2Paddle)工具转换MobileNetV1预测模型步骤：
+| MobileNetV1 | Baseline | 70.90 | - | 33.15 |
+| MobileNetV1 | 量化+蒸馏 | 70.49 | - | 13.64 |
+| ResNet50_vd | Baseline | 79.12 | 3.1901 | - |
```
Comment (wanghaoshuang) [inline demo/auto_compression/image_classification/README.md]: 小数点后统一保留2位

**Review by wanghaoshuang — APPROVED** [review]
